synapse/app/homeserver.py

```diff
@@ -415,6 +420,10 @@ def start():
 
                 # Check if it needs to be reprovisioned every day.
                 hs.get_clock().looping_call(reprovision_acme, 24 * 60 * 60 * 1000)
+            if hs.config.oidc_enabled:
```

Comment:
Please add a comment above this about what's being configured (and probably leave an empty line from the section above this).